packages/create-bison-app/template/src/server/routers/user.ts

```diff
@@ -23,7 +23,7 @@ export const userRouter = t.router({
     .input(z.object({ id: z.string().optional(), email: z.string().optional() }).optional())
     .query(({ ctx, input }) => ctx.db.user.findMany({ where: input, select: defaultUserSelect })),
   find: t.procedure
-    .input(z.object({ id: z.string().optional(), email: z.string().optional() }))
+    .input(z.union([z.object({ id: z.string() }), z.object({ email: z.string() })]))
```

Comment:
This change requires either the `id` or `email` to be given.

Fixes this error after upgrading Prisma:
<img width="837" alt="Screen Shot 2023-08-18 at 4 17 10 PM" src="https://github.com/echobind/bisonapp/assets/1087679/038aab51-a0cf-4ac0-bf78-f6ff46ab6b82">

Reply:
Nice use of `z.union`